paasta_tools/autoscaling_lib.py

```diff
@@ -610,8 +611,8 @@ def wait_and_terminate(slave, dry_run):
                 if not instance_id:
                     log.warning("Didn't find instance ID for slave: {0}. Skipping terminating".format(slave['pid']))
                     continue
-                is_ready_to_kill = True
-                # is_ready_to_kill = paasta_maintenance.something(slave['pid'])
+                # Check if no tasks are running or we have reached the maintenance window
+                is_ready_to_kill = is_safe_to_kill(slave['hostname'])
                 if is_ready_to_kill:
```

Comment:
Why not just do `if is_safe_to_kill(slave['hostname'])` to avoid having the duplicate terminology (`is_ready_to_kill` and `is_safe_to_kill`)?